src/SerialLoops/EditorTabsPanel.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Eto.Drawing;
+using Eto.Forms;
+using SerialLoops.Editors;
+using SerialLoops.Lib;
+using SerialLoops.Lib.Items;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using static System.Windows.Forms.VisualStyles.VisualStyleElement;
+
+namespace SerialLoops
+{
+    internal class EditorTabsPanel : Panel
+    {
+
+        public static readonly Size EDITOR_BASE_SIZE = new(500, 420);
+
+        public TabControl Tabs { get; private set; }
+        
+        private readonly Project _project;
+
+        public EditorTabsPanel(Project project)
+        {
+            _project = project;
+            InitializeComponent();
+        }
+
+        void InitializeComponent()
+        {
+            MinimumSize = EDITOR_BASE_SIZE;
+            Padding = 0;
+
+            Tabs = new();
+            Content = new StackLayout
+            {
+                Orientation = Orientation.Vertical,
+                MinimumSize = EDITOR_BASE_SIZE,
+                Items =
+                {
+                    Tabs
+                }
+            };
+        }
+
+        public static Icon GetItemIcon(ItemDescription.ItemType type)
+        {
+            //todo battle with this and figure out loading from a resx?
+            //return Icon.FromResource($"SerialLoops.Icons.{type}.png", Assembly.GetExecutingAssembly());
+            throw new NotImplementedException();
+        }
+
+        internal void OpenTab(ItemDescription item)
+        {
+            // If a tab page with the name and type exists, switch to it
+            foreach (TabPage page in Tabs.Pages)
+            {
+                if (page.Text.Equals(item.Name))
```

Comment:
for readability, i'd still just do basic comparison here:

```suggestion
                if (page.Text == item.Name)
```

Reply:
The only time I'd use `.Equals` would be if I were planning to use one of its other arguments, such as `StringComparison.OrdinalIgnoreCase`.